app/annotationsList/annotationList.js

```diff
@@ -75,11 +75,21 @@ app.controller('AnnotationListCtrl', function($rootScope, $scope, $http, $uibMod
   }
 
   function postProcess() {
```

Comment:
The problem with this approach is that if one of the APIs is down or slow it takes longer to show the information on the page. It would be much better for that data to be bound asynchronously in the table.